command/clean/command.go

```diff
@@ -0,0 +1,29 @@
+package clean
+
+import (
+	"flag"
+	"github.com/tueftler/doget/command"
+	"github.com/tueftler/doget/config"
+	"github.com/tueftler/doget/dockerfile"
+	"os"
+)
+
+// CleanCommand allows to remove the vendor directory and all of its contents
+type CleanCommand struct {
+	command.Command
+	flags *flag.FlagSet
+}
+
+// NewCommand creates new clean command instance
+func NewCommand(name string) *CleanCommand {
+	return &CleanCommand{flags: flag.NewFlagSet(name, flag.ExitOnError)}
+}
+
+// Run performs action of clean command
+func (c *CleanCommand) Run(parser *dockerfile.Parser, args []string) error {
+	if _, ok := os.Stat(config.Vendordir); nil == ok {
```

Comment:
I find this a bit confusing, since os.Stat returns `FileInfo, error`. Why didn't you use `err` as variable name?